exporter/elasticexporter/apmserverexporter.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package elasticexporter
```

Comment:
```suggestion
package elasticapmexporter
```

Reply:
As I have explained in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/2198#issuecomment-776264188 , please note that 
* The `elasticexporter` has been here for 8 months and relates to Elastic as a company and to the Elastic Observability solution
* We have not seen any misunderstanding of people who use a tracing solution that store its traces in Elasticsearch (e.g. Jaeger or Zipkin) and who would expect the Elastic Exporter to be what they need. We will be happy to update the [Elastic exporter docs](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/elasticexporter) if we see confusion.